test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
+
+LinkAuthor = Class.new(Model)
+class LinkAuthorSerializer < ActiveModel::Serializer
+  links do
+    link :self, href: "//link_author/#{object.id}"
```

Comment:
> those are arbitrary strings anyways.

That's exactly why I don't want the links method.  Let's just let `links` from the controller, then, in another PR, build a `ApiObject::Link` and use that. Then, if someone comes up with a good use case for the serializer, we'll add it then